bigbluebutton-html5/imports/ui/components/join-handler/presenceManager/component.tsx

```diff
@@ -162,9 +163,13 @@ const PresenceManager: React.FC<PresenceManagerProps> = ({
 
   const errorCode = loggedOut ? 'user_logged_out_reason' : joinErrorCode || ejectReasonCode;
 
+  const userCurrentlyInMeeting = allowToRender && !(meetingEnded || joinErrorCode || ejectReasonCode || loggedOut);
+
+  updateMeetingStatusForPlugin(userCurrentlyInMeeting);
```

Comment:
I don't like this approach on bbb, normally we use components to be listeners, like [notifications](https://github.com/bigbluebutton/bigbluebutton/blob/16c42360c5af4d3e1adb2ab6900124a4cc8ac958/bigbluebutton-html5/imports/ui/components/notifications/component.tsx) or [connection status](https://github.com/bigbluebutton/bigbluebutton/blob/3d5c33129a1f698aaabd62761cb6bee564f15481/bigbluebutton-html5/imports/ui/components/connection-status/component.jsx)